docs/usage_guide.rst

```diff
@@ -731,6 +731,59 @@ rotki automatically resolves ens name for each of your ethereum accounts. If the
    :alt: Example of resolving an ens name
    :align: center
 
+
+Ethereum Transactions
+=====================
+
+rotki is capable of getting and decode (understand what is hapening) in your ethereum transactions. When you visit the ``Ethereum Transactions`` sections the process to obtain all the information will start. You will be able to check
+the status in a informative breakdown per address
+
+.. image:: images/eth_tx_query_process.png
+   :alt: Ethereum transactions query status breakdown
+   :align: center
+
+It is possible that you need to redecode some transactions if the set of decoding rules is modified. To redecode a transaction you have two possibilities. The first of them is to click on the three dots to display the options for a ethereum transaction and clcik on ``Redecode events``. This will start the process to read again the transaction's events and try to understand what happened in them.
```

Comment:
```suggestion
It is possible that you need to redecode some transactions if the set of decoding rules is modified. To redecode a transaction you have two possibilities. The first of them is to click on the three dots to display the options for an ethereum transaction and click on ``Redecode events``. This will start the process to read the transaction's events again and try to understand what happened in them.
```